.github/workflows/tpu-tests.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       (startsWith(github.event_name, 'pull_request') && contains(github.event.pull_request.title, 'TPU'))
     strategy:
       fail-fast: false
+      max-parallel: 2
```

Comment:
OK, I am fine with playing the lottery further...

Reply:
yes, so this is correct, it is a much higher chance to all pass if we run just two at the time then requesting 4 at the same time...

Reply:
When debugging with these jobs, you want them to run ASAP so that either they fail fast and let you re-run through the GitHub interface, or they simply run as normal

Doing this change means that out of the 4 jobs that we run, you have to wait until the first two start for you to see what happens on the second two. It's annoying

Please believe me, as someone who runs these jobs, this change is not helpful, if anything annoying as it adds unnecessary waiting

Reply:
Please believe me that I am suffering from this failure; not a TPU-related change but requires TU and needs to be rerun 2-4 times to get it passed. It seems fewer jobs in parallel help... so when you last time to debug it in CI and not on a real machine?

Reply:
On every single TPU feature PR.

> not a TPU-related change but requires TU

This is false, TPUs tests are only required if you touch the TPU workflow: https://github.com/Lightning-AI/lightning/blob/master/.github/checkgroup.yml#L92-L95

If you touch the TPU workflow, then it must be a TPU-related change

Reply:
cool, with this logic, you need to agree that touching docs in any way shall be labeled as docs change :chipmunk: